### PR TITLE
Add instructions on creating a secret

### PR DESCRIPTION
**Pull request by Mohammed-Shoaib**
Hey,

As per the discussion on #9, I added user instructions on creating a secret :key: for the environment variable `GH_PAT` :sparkles:.

Let me know for any further changes, Thanks! :unicorn:

Best,
Mohammed Shoaib

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,12 +42,15 @@ jobs:
...
+2. Create a [secret](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets) 🔑 by going to **GitHub repo > Settings > Secrets > New secret** with the following:
+- Name: `GH_PAT`.
+- Value: The token with the `gist` scope generated previously.
+
+3. 💰 Profit
 
 ### Environment variables & secrets
 
 - **GIST_ID:** The ID portion from your gist url `https://gist.github.com/matchai/`**`6d5f84419863089a167387da62dd7081`**.
-- **GH_PAT:** The GitHub token generated above.
```
Comment (JasonEtco) [inline README.md:50]: Can we keep this in? It's still a helpful reference for the requirements of the `.yml` file.

File: README.md [hunk 2]
```diff
@@ -42,12 +42,15 @@ jobs:
           GIST_ID: 123abc
 ```
 
-2. 💰 Profit
+2. Create a [secret](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets) 🔑 by going to **GitHub repo > Settings > Secrets > New secret** with the following:
```
Comment (JasonEtco) [inline README.md:45]: Tiny tweak:

```suggestion
2. [Create a secret](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets) 🔑 by going to **GitHub repo > Settings > Secrets > New secret** with the following:
```

File: README.md (hunk 1 above)
Comment (Mohammed-Shoaib) [reply README.md]: Alright, sounds good to me!

File: README.md (hunk 2 above)
Comment (Mohammed-Shoaib) [reply README.md]: This tiny tweak makes it look so much better.

**Comment by Mohammed-Shoaib** [comment]
Thank you for the great suggestions! I really appreciate your time and effort into looking at every little detail no matter how small it may be, Thank you so much! :relaxed:

**Pull request merged**